src/kernels/jupyter/serverSelector.ts

```diff
@@ -72,6 +72,57 @@ export type SelectJupyterUriCommandSource =
     | 'errorHandler'
     | 'prompt';
 
+export async function validateSelectJupyterURI(
```

Comment:
I thought about implementing everything completely in `standalone` but that would be quite some effort so for now I'm still using helper functions from the core.